build/wix/main.wxs

```diff
@@ -67,8 +69,45 @@
         </Custom>
       </InstallExecuteSequence>
 
+      <!-- Setting the PATH variable -->
+      <Component Id="PathUser" Directory="APPLICATIONFOLDER">
+        <Condition>MSIINSTALLPERUSER</Condition>
+        <RegistryValue
+          Root="HKCU"
+          Key="SOFTWARE\!(wix.AppGUID)"
+          Name="EnvironmentVariablesSet"
+          Value="yes"
+          Type="string"
+          KeyPath="yes"
+        />
+        <Environment Id="PathWindowsUser" Name="PATH"
+          Action="set" Part="last" System="no" Permanent="no"
+          Value="[APPLICATIONFOLDER]resources\resources\win32\bin\" />
```

Comment:
Should the paths be declared as variables so that they can be reused in both `PathUser` and `PathSystem`?

Reply:
While that's possible, I think that will make things harder to understand; that layer of indirection doesn't seem necessary (especially since we are probably not going to add more directories, in the short term).

I'd still be fine with adding a property for this if you really prefer, though.

Reply:
Cool, thanks for clarifying. Let's leave it as is if there is no more declaration.